cmd/test-server/main.go

```diff
@@ -67,5 +66,5 @@ func main() {
 			log.Fatalf("ListenAndServe port 80 error: %v", err)
 		}
 	}()
-	http.ListenAndServeTLS(":443", fullChain, privKey, nil)
+	log.Fatal(http.ListenAndServeTLS(":443", fullChain, privKey, nil))
```

Comment:
Should this error also be logged into ErrLogger? Otherwise it is printed to standard error (which I am storing for now).
Or should we be writing Fatal errors to stderr?

Reply:
Fixed in 6471c18c38447c7bd56aa218fe6fa4f42c38cfc0.